packit_service/models.py

```diff
@@ -461,10 +461,19 @@ class GitProjectModel(Base):
     project_url = Column(String)
     instance_url = Column(String, nullable=False)
 
+    # we checked that exists at least a bodhi update or a koji build
+    # or a merged packit downstream pull request for it.
+    onboarded = Column(Boolean, default=False)
```

Comment:
since this is specific to downstream Fedora automation, I would somehow explicitly mention this in the name (onboarded_downstream or similar)